debian/changelog

```diff
@@ -1,3 +1,66 @@
+mir (2.17.0) UNRELEASED; urgency=medium
+
+  [ Matthew Kosarek ]
+  * New upstream release 2.17.0
+
+    - ABI summary:
+      . miral ABI bumped to 7
+      . mircommon ABI bumped to 10
+      . mircookie has been removed
+      . mircore ABI unchanged at 2
+      . miroil ABI bumped to 5
+      . mirplatform ABI bumped to 28
+      . mirserver ABI bumped to 60
+      . mirwayland ABI bumped to 5
+      . mirplatformgraphics ABI bumped to 22
+      . mirinputplatform ABI numed to 9
+
+    - Enhancements:
+      . [platforms] Much better display and graphics platform selection, especially for hybrid and Nvidia systems. (PR: #3220)
+      . [input] Add HW cursor support. (Fixes #3198)
+      . [miral] Add support for logind's lock and unlock signals so that compositors may implement some behavior in response to these signals, like showing a lockscreen (PR: #3232)
+      . [Wayland] wlr foreign toplevel management protocol returns the desktop id as the app_id if it can. (PR: #3147)
+      . [window management] The floating window manager places new windows at an offset from the previously opened window. (Fixes #3175)
+      . [Wayland] Prototype an implementation of the mir-shell protocol (PR: #3166)
+      . [Wayland] Add support for zwlr_layer_surface_v1::set_exclusive_zone(-1) (Fixes: #1992)
+      . [Wayland] Add support for wl_surface enter/leave events. (Fixes: #342)
+      . [miral] Provide a mechanism to allow 3rd-party compositors to override the renderer in miral. (PR: #3338)
+      . [mirserver] Provide access to the Surface from the Renderable. (PR: #3355)
+      . Add support for floating point commandline options. (PR: #3324)
+      . Build mir-libs for both core22 and core24. (PR: #3364)
```

Comment:
No need to change this now; but, for the future, I don't mention infrastructure related changes in the changelog as they are not interesting to downstreams.

Reply:
We can remove this if you want to here! I was wondering if they were important too

Reply:
FYI, this should be good to look at. I just have to sign the final commit and whatnot, but the changelog should be good to go

Reply:
_Do not sign the commit until testing has been completed,_ that starts off the release automation.

(We haven't even finished the [PPA builds](https://launchpad.net/~mir-team/+archive/ubuntu/rc/+packages) yet)

Reply:
Ah okay :)